src/portal-impl.c

```diff
@@ -316,65 +369,48 @@ load_portal_configuration (gboolean opt_verbose)
   if (portal_dir == NULL)
     portal_dir = DATADIR "/xdg-desktop-portal";
 
-  conf = g_new0 (PortalConfig, 1);
+  user_portal_dir = g_build_filename (g_get_user_config_dir (),
```

Comment:
For system config the data_dir is used, for symmetry I think get_user_data_dir would make more sense.

Reply:
There's also the option of using sysconfdir for the system configuration. I am fine with either.

Reply:
I switch to using `sysconfdir`, as it does make much more sense.

There's also the option of using:

- `sysconfdir` for site configuration
- `prefix` / `sysconfdir` for packaging configuration